packages/@uppy/companion/infra/kube/gcloud-deploy.sh

```diff
@@ -7,33 +7,32 @@ set -o nounset
 # Set magic variables for current FILE & DIR
 __dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 __kube="${__dir}"
+__companion="$(dirname "$(dirname "${__kube}")")"
+# Install kubectl
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
```

Comment:
Shall we pin the currently used kubectl and hence make version bumps opt-in vs organic?